.ddqa/config.toml

```diff
@@ -28,6 +31,9 @@ jira_statuses = [
   "Done",
 ]
 github_team = "platform-integrations"
+exclude_members = [
+  "olivielpeau",
+]
```

Comment:
Julia is also in Platform Integration, I think we can add her here too, wdyt? (https://github.com/orgs/DataDog/teams/platform-integrations)

Reply:
Fixed, thanks! I had copied from Agent Platform by mistake